Examples/drawing-tests.lisp

```diff
@@ -475,7 +481,8 @@
 	    (with-drawing-options (stream :line-thickness lt :line-joint-shape lj :line-dashes ld)
 	      (draw-text* stream (format nil "~A ~A ~A" ld lj lt) 20 y)
 	      (let ((x-positions '(150 300)))
-		  (draw-rectangle* stream 200 (+ 10 y) 450 (+ 35 y)
+                (declare (ignore x-positions))
+                (draw-rectangle* stream 200 (+ 10 y) 450 (+ 35 y)
 				   :filled nil))
```

Comment:
if x-position is ignored, we may get rid of a whole let and move body one level up, right?